src/openforms/emails/digest.py

```diff
@@ -366,7 +368,33 @@ def collect_invalid_registration_backends() -> list[InvalidRegistrationBackend]:
     return invalid_registration_backends
 
 
+def introspect_json_logic_wrapper(
+    expression: JSON, form_name: str
+) -> list[InputVar] | None:
+    try:
+        introspection_result = introspect_json_logic(expression).get_input_keys()
+    except Exception as e:
+        logger.info(
+            "exception in json logic (rule - action) of form %s: %s",
+            form_name,
+            str(e),
+        )
```

Comment:
```suggestion
        logger.error(
            "malformed/unsupported JsonLogic expression in form %s: %r",
            exprssion,
            form_name,
            exc_info=e,
        )
```
logging is an art :) but if you pass the exception via `exc_info`, then that will be added to the traceback/logging infrastructure and Sentry for example should show the exception information.

Additionally, since this causes a proper crash, a log level of `error` is not too much. By default `warning` and higher end up in Sentry.

Stringified exceptions in log messages tend not to be too useful, especially because causes of exceptions get lost (if you do `raise X from Y`, the `from Y` information gets lost I think, which doesn't happen with the `exc_info=e` mechanism.